### ISSUE TITLE
Preference option for "font size" (RM #4591)

### ISSUE THREAD

**Issue by dpage** [pgadmin-org/pgadmin4#2868]
***Issue migrated from Redmine: https://redmine.postgresql.org/issues/4591***
*Originally created by **Stuart Green** at 2019-08-09 11:08:06 UTC.*

It would be great to have an option within File->Preferences dialog to set a global font-size. At the moment I find the fonts too large, especially compared to pgAdmin 3, but this is rather subjective.

Simplest would be to set a global font-size for "body", which adjusts the other relative-sized fonts within the CSS. I guess a more complex implementation would allow specific font-sizes for the SQL editor panel and others.

Thanks for your hard work.

**Comment by dpdoughe** [comment]
Yes I noticed after upgrading to v7.3 (Windows) that the font sizes are very large. The only way I've found to get back to normal font size is to File->Reset Layout. The preferences don't seem to exist to set a default font size. How does all of this work? Why do I need to Reset Layout? If there is a way to set the font size explicitly, it certainly isn't obvious or intuitive.

**Comment by adityatoshniwal** [comment]
@dpdoughe There is an option to reset the font size. File->Runtime->Actual Size.

**Comment by dpdoughe** [comment]
Thank you @adityatoshniwal .  It does seem curious location for font size preferences. File -> Runtime isn't where I think most users will expect to see Font size options when there is a File -> Preferences option already with other frontend prefs there 🤔

**Comment by dpage** [comment]
Closing as part of a housekeeping pass on issues migrated from the old Redmine tracker. This issue has had no activity of note since the migration, and because the migration attributed authorship to me, the original reporter won't be notified of any updates here. If this is still relevant, please feel free to open a fresh issue.
